indico/modules/rb/util.py

```diff
@@ -334,6 +334,7 @@ def get_prebooking_collisions(reservation):
 
 
 def check_impossible_repetition(data):
+    """ Check if the repetition is possible with the given data """
```

Comment:
no surrounding spaces please ;)

also, this docstring is just a more verbose form (did github copilot write it? :P) of the function name...

how about something like this instead:

```
"""
Check for broken repetition data

This checks that a repetition using weekdays has a date range
containing at least one of those weekdays.
"""
```